client/component/comment.js

```diff
@@ -401,9 +412,11 @@ export function Composer({about=null, commentKey, goBackAfterPost=false, topLeve
             onCancel={goBackAfterPost && onCancel} />
         <Byline type='large' userId={personaKey} subtitleLabel={subtitle} />
         <Pad size={24} />
-        <EditComment big comment={editedComment ?? comment ?? {text: ''}} topLevel={topLevel}
-            onCancel={goBackAfterPost && onCancel}
-            setComment={setEditedComment} onEditingDone={onEditingDone} />
+        <FocusProvider>
```

Comment:
Rather than having a context for this, can we just have TextField have an optional onFocusChange callback, than EditComment plugs into?

Reply:
done